modules/exploits/windows/imap/mercury_login.rb

```diff
@@ -4,87 +4,83 @@
 ##
 
 class MetasploitModule < Msf::Exploit::Remote
-  Rank = AverageRanking
+  Rank = NormalRanking
 
   include Msf::Exploit::Remote::Tcp
+  include Msf::Exploit::Remote::Seh
+  include Msf::Exploit::Remote::Egghunter
 
   def initialize(info = {})
     super(update_info(info,
-      'Name'           => 'Mercury/32 LOGIN Buffer Overflow',
-      'Description'    => %q{
+      'Name' => 'Mercury/32 4.01 IMAP LOGIN SEH Buffer Overflow',
+      'Description' => '
```

Comment:
`%q{ }` notation is preferred for multi-line module descriptions.

Reply:
will fix. thanks for patience, first commit